src/components/task-plan/reading.cjsx

```diff
@@ -116,10 +116,20 @@ ReadingPlan = React.createClass
 
   render: ->
     id = @getParams().id
-    footer = <BS.Button bsStyle="primary" onClick={@publish}>Publish</BS.Button>
+
+    if id
+      plan = TaskPlanStore.get(id)    
+    else
+      plan = TaskPlanStore.get(@state.id)
+     
+    if plan.title and plan.due_at and plan.settings.page_ids.length > 0
+    then btnDisable = '' else btnDisable = 'disabled'
```

Comment:
ok I will update syntax.  Also I noticed this module may be helpful : https://github.com/JedWatson/classnames

It was originally here : http://facebook.github.io/react/docs/class-name-manipulation.html